azkaban-common/src/main/java/azkaban/flow/FlowExecutionType.java

```diff
@@ -0,0 +1,27 @@
+package azkaban.flow;
+
+public enum FlowExecutionType {
+  SCHEDULED("scheduled"), ADHOC("adhoc");
+
+  private final String value;
```

Comment:
If you create enum in database with SCHEDULED and ADHOC as values, there is no need of value instance variable. Name of enum will be enough.

Reply:
+1. Came to say exactly this.